src/KubernetesClient.Aot/KubernetesJson.cs

```diff
@@ -91,6 +91,11 @@ public static TValue Deserialize<TValue>(Stream json, JsonSerializerOptions json
 
         public static string Serialize(object value, JsonSerializerOptions jsonSerializerOptions = null)
         {
+            if (value is V1Patch { Content: string jsonValue })
```

Comment:
i did not get it, is it still some dynamic thing?

Reply:
I don't quite understand the question?

In the usages I've seen, the Content property of the V1Patch object is always set as the json string value of the patch request body. This pattern, broken down, checks:

value is of type V1Patch, and is not null
value.Content is of type string
Then it assigns the value of value.Content to jsonValue, however this is all essentially just syntactic sugar, so all of that is resolved at compile time into a slightly uglier string of conditions.

Reply:
let me ask this way?
is it a breaking change or different behavior from non-aot version?
the assumption is that V1Patch must be json string?

Reply:
Ahhh, got you, this isn't a breaking behaviour, and I would say it's restoring some functionality that exists in the non-aot version to the aot version. 

I don't think full parity can be achieved without changing the API surface.

Reply:
could you please also provide an example of aot json, thne i think i am good with this pr

Reply:
I've added that, mostly copying the existing patch example